src/message/messageActionSheet.js

```diff
@@ -189,6 +194,7 @@ const actionSheetButtons /* ActionSheetButtonType[] */ = [
   { title: 'Reply', onPress: reply, onlyIf: isSentMessage },
   { title: 'Copy to clipboard', onPress: copyToClipboard, onlyIf: isNotDeleted },
   { title: 'Share', onPress: shareMessage, onlyIf: isNotDeleted },
+  { title: 'Add reaction', onPress: addReaction, onlyIf: isNotDeleted },
```

Comment:
Use `resolveMultiple ` here, we we don't want want this option when message is in outbox.